diff -up knotifications-5.87.0/KF5NotificationsConfig.cmake.in.omv~ knotifications-5.87.0/KF5NotificationsConfig.cmake.in
--- knotifications-5.87.0/KF5NotificationsConfig.cmake.in.omv~	2021-10-15 10:51:46.251748841 +0200
+++ knotifications-5.87.0/KF5NotificationsConfig.cmake.in	2021-10-15 10:52:15.501959233 +0200
@@ -19,11 +19,6 @@ if (NOT @BUILD_SHARED_LIBS@)
         find_dependency(KF5WindowSystem "@KF_DEP_VERSION@")
     endif()
 
-    if (@WITH_SNORETOAST@)
-        find_dependency(LibSnoreToast)
-        find_dependency(Qt5Network @REQUIRED_QT_VERSION@)
-    endif()
-
     if (@HAVE_PHONON4QT5@)
         find_dependency(Phonon4Qt5)
     endif()
@@ -40,17 +35,10 @@ if (NOT @BUILD_SHARED_LIBS@)
         find_dependency(Qt5X11Extras @REQUIRED_QT_VERSION@)
     endif()
 
-    if(APPLE)
-        find_dependency(Qt5MacExtras @REQUIRED_QT_VERSION@)
-    endif()
-
     if(@HAVE_DBUSMENUQT@)
         find_dependency(dbusmenu-qt5)
     endif()
 
-    if (ANDROID)
-        find_dependency(Qt5AndroidExtras @REQUIRED_QT_VERSION@)
-    endif()
 endif()
 
 include("${CMAKE_CURRENT_LIST_DIR}/KF5NotificationsTargets.cmake")
